README.md

```diff
@@ -5,7 +5,7 @@
   <img src="./assets/murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-<h3><b>Microverse Capstone Project - Module 1</b></h3>
+<h3><b>Leaderboard Project</b></h3>
 
```

Comment:
- [ ] In this readme doc, kindly include the Future Feature section as we have in the original readme template

- [ ]  In the Getting Started section of the readme doc kindly fix your link here:

  ![image](https://user-images.githubusercontent.com/54848363/208676175-d03a6632-e3ce-476b-88b5-46cc183e47dc.png)